test/quicklink.spec.js

```diff
@@ -34,10 +34,10 @@ mainSuite('should prefetch in-viewport links correctly (UMD)', async () => {
   });
   await page.goto(`${server}/test-basic-usage.html`);
   await sleep(1000);
-  assert.equal(Array.isArray(responseURLs), true);
-  assert.equal(responseURLs.includes(`${server}/1.html`), true);
-  assert.equal(responseURLs.includes(`${server}/2.html`), true);
-  assert.equal(responseURLs.includes(`${server}/3.html`), true);
+  assert.is(Array.isArray(responseURLs), true);
+  assert.is(responseURLs.includes(`${server}/1.html`), true);
+  assert.is(responseURLs.includes(`${server}/2.html`), true);
+  assert.is(responseURLs.includes(`${server}/3.html`), true);
```

Comment:
low prio: These can be `assert.ok` which checks truthiness
```suggestion
  assert.ok(Array.isArray(responseURLs));
  assert.ok(responseURLs.includes(`${server}/1.html`));
  assert.ok(responseURLs.includes(`${server}/2.html`));
  assert.ok(responseURLs.includes(`${server}/3.html`));
```